armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -740,6 +758,22 @@ DNSSEC=yes
 Cache=yes
 EOF
 
+cat << EOF > /etc/network/interfaces
```

Comment:
Do we need to override `/etc/network/interfaces`? On the `eve` node, built from a recent commit on `master`, it seems to already exist and have the equivalent content as the file you're creating here:

```
root@eve:~# cat /etc/network/interfaces
source /etc/network/interfaces.d/*
# Network is managed by Network manager
auto lo
iface lo inet loopback
```

Reply:
I'll try if it just works like that.

Reply:
Removed the explicit `/etc/network/interfaces`, works just fine.